api/v1beta1/cloudstackcluster_webhook.go

```diff
@@ -43,7 +44,7 @@ var _ webhook.Defaulter = &CloudStackCluster{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *CloudStackCluster) Default() {
-	cloudstackclusterlog.Info("default", "name", r.Name)
+	cloudstackclusterlog.V(1).Info("entered api default setting webhook", "api resource name", r.Name)
```

Comment:
These log lines seemed rather noisy. Bumped to V(1)

Reply:
how do you set the verbosity when running CAPC?